mysql/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM eventuateio/eventuate-tram-mysql:0.3.0.RELEASE
```

Comment:
I suspect that you don't need to build an image for the project. There is no custom schema

Reply:
I will check